### PR TITLE
Buttons with variant

### PR DESCRIPTION
**Pull request by tayfuntayfun**

### CODE REVIEW

File: src/components/Button/index.js [hunk 1]
```diff
@@ -1,252 +1,111 @@
...
+    {args.label}
+  </StyledDangerButton>
+);
 
-export default Button;
+export const OutlinedButton = ({ ...args }) => (
+  <StyledOutlinedButton background={args.backgroundColor} color={args.color}>
+    {args.label}
+  </StyledOutlinedButton>
+);
```
Comment (KrustyC) [inline src/components/Button/index.js:111]: I like the implementation, but as we are using styled components you can export a single Button component and accept a property called, for instance, "variant" which cant take 4 different values ('primary', 'secondary', 'danger', 'outlined'), so then you can use the button as so 
```
<Button variant="primary>
   Click Here
</Button>
```

Inside the component you can then change implementation for each type as such
```
export const Button = styled.button`
  ${({ variant }) => css`
      color: #ffffff;
      background-color: #dc3545;
      font-weight: 400;
      text-align: center;
  `}
  top: 1432px;
  color: #ffffff;
  background-color: #dc3545;
  font-weight: 400;
  text-align: center;
  white-space: nowrap;
  vertical-align: middle;
   
   ${variant === 'secondary' && css`
       // Here the style that differs in the secondary
   `}

    ${variant === 'danger' && css`
       // Here the style that differ in the secondary
   `}

}`;
```

**Review by KrustyC — CHANGES_REQUESTED** [review]

File: src/components/Button/index.js (hunk 1 above)
Comment (KrustyC) [reply src/components/Button/index.js:111]: By doing so, you will not have to replicate common styles between different buttons, and also you will be able to export a single component where user can change the style by simply updating the value of variant

Comment (tayfuntayfun) [reply src/components/Button/index.js]: Thanks Davidec, really useful stuff. I made the changes as you advised. Thanks

Comment (tayfuntayfun) [reply src/components/Button/index.js]: I refactored the code . Removed controllers and cleaned unnecessary parts.

**Review by KrustyC — APPROVED** [review]

**Pull request merged**